### PR TITLE
[#52] Define basic CSS for route result component

### PR DESCRIPTION
**Pull request by mdelapenya**
Basic styles for results components

### CODE REVIEW

**Comment by mdelapenya** [comment]
Assigning to @felixortegam, but @javierlopezdeancos could take a :eyes:  too...

**Comment by felixortegam** [comment]
Looks good to me :+1: waiting for @javierlopezdeancos  :)

**Comment by mdelapenya** [comment]
Anything I can do for this PR?

File: src/components/results/bus-stop-result/_bus-stop-result.scss [hunk 1]
```diff
@@ -1,13 +1,21 @@
-.#{$namespace}-bus-stop-result {
+.#{$namespace}-bus-stop-results {
     display: flex;
     flex-direction: column;
     align-items: center;
+    width: 100%;
 
-    &__header {
-        margin-bottom: 8vh;
+    &__bus-stop-description {
```
Comment (javierlopezdeancos) [inline src/components/results/bus-stop-result/_bus-stop-result.scss:7]: i not correct

the correct is &__description

review my fork with the new structure and naming for results components

**Comment by javierlopezdeancos** [comment]
you need merge he last changes in my fork in the feature-include-redux branch

because the structure is change to get better way with redux

**Comment by mdelapenya** [comment]
ok, will resend then

Thanks!

**Pull request closed without merging**